web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts

```diff
@@ -109,10 +109,20 @@ export class SubmissionPanelComponent implements OnInit, OnDestroy {
 
   getCaptureLocationCoord(task: Task): string {
     // x represents longitude, y represents latitude
-    const {x, y} = (this.getTaskSubmissionResult(task)!.value as Point).coord;
+    const {coord, accuracy, altitude} = this.getTaskSubmissionResult(task)!
+      .value as Point;
+    const {x, y} = coord;
     const long = Math.abs(x).toString() + (x > 0 ? '° E' : '° W');
     const lat = Math.abs(y).toString() + (y > 0 ? '° N' : '° S');
-    return lat + ', ' + long;
+
+    let result = lat + ', ' + long;
```

Comment:
Consider using string templates here and below instead

Reply:
see https://github.com/google/ground-platform/pull/2102